lib/ui/window/platform_configuration.h

```diff
@@ -633,6 +634,11 @@ class PlatformConfigurationNativeApi {
 
   static void RegisterBackgroundIsolate(int64_t root_isolate_token);
 
+  static Dart_Handle AddPlatformPortCallback(const std::string& name,
+                                             Dart_Handle send_port);
+
+  static Dart_Handle RemovePlatformPortCallback(const std::string& name);
```

Comment:
Can you not remove this API and have someone call "SetChannelPort(name, Dart_Null()); ?